sugarlabs/www
# Fix favicon "not found" error on non-homepage pages

This PR addresses the issue where favicons were not found on non-homepage pages, particularly on Firefox browser. The problem was traced to an incorrect favicon path that was not being handled properly across different pages.

### Changes Made:
- Updated the favicon link in `_layouts/base.html` to use a consistent, absolute path.
- Modified `js/main.js` to dynamically update the favicon based on a random color scheme.
- Implemented error handling in the JavaScript to revert to a default favicon if the specific one is not found.

### Issue Link:
Closes #507

### How to Test:
1. Load the homepage and any non-homepage page (e.g., `/about-us` or `/press`) on Firefox.
2. Check if the favicon is displayed correctly across all pages.
3. Ensure that a fallback favicon is shown if the dynamic one is missing.

### Additional Notes:
- This issue was specifically observed in the Firefox browser.
- The fix ensures consistent favicon display on all pages, improving the site's visual consistency and user experience.

### OS and Browser Information:
- **Operating System**: Linux
- **Browser**: Firefox
